sw/device/boot_rom/boot_rom.c

```diff
@@ -32,6 +33,9 @@ static dif_uart_t uart0;
 void _boot_start(void) {
   test_status_set(kTestStatusInBootRom);
   pinmux_init();
+  flash_init();
```

Comment:
@cfrantz @moidx 
in general how do you guys like to handle this kind of polling in ROM? 
should we have some kind of timeout?

Reply:
I think we may want to implement a timeout so that we can report status and shutdown.  There is not much we can do if flash scrambling fails right?

Reply:
there is not.  So reporting status and shutdown would probably be good.

Reply:
let me file a separate issue on how timeouts should be handled in ROM then.